events-api/src/modules/events/events.repository.ts

```diff
@@ -18,4 +21,156 @@ export class EventsRepository {
 
     return result;
   }
+
+  static async getOne(eventId: string): Promise<EventModel[]> {
```

Comment:
Method is called getOne, but array is returned. Seems a bit contradictory for me